chia/full_node/full_node_rpc_api.py

```diff
@@ -98,6 +101,7 @@ def __init__(self, service: FullNode) -> None:
         self.service = service
         self.service_name = "chia_full_node"
         self.cached_blockchain_state: Optional[dict[str, Any]] = None
+        self.executor = ThreadPoolExecutor(max_workers=2, thread_name_prefix="node-rpc-")
```

Comment:
this doesn't seem to improve throughput of these RPCs. It's not clear to me why, perhaps there's only a single `asyncio.Task` per RPC connection. In either case, compute heavy work should not be performed on the main thread.